frontend/src/components/common/Resource/Resource.tsx

```diff
@@ -440,7 +444,7 @@ function LivenessProbes(props: { liveness: KubeContainer['livenessProbe'] }) {
   return (
     <Box display="flex" flexDirection="column">
       <LivenessProbeItem>
-        {`http-get, path: ${liveness?.httpGet?.path}, port: ${liveness?.httpGet?.port}, 
+        {`http-get, path: ${liveness?.httpGet?.path}, port: ${liveness?.httpGet?.port},
```

Comment:
do we need this change here??

Reply:
Not sure. It does have a trailing whitespace that should not be there, but it's not part of this patch. Thanks.